### ISSUE TITLE
MSRE permission denials

### ISSUE THREAD

**Issue by shawniverson** [E-F-A/v4#173]
https://forum.efa-project.org/viewtopic.php?f=19&t=4289&p=15888

**Comment by shawniverson** [comment]
```
type=AVC msg=audit(1595192902.993:1064): avc:  denied  { add_name } for  pid=1258 comm="php-fpm" name="sig.html.rules.bak" scontext=system_u:system_r:httpd_t:s0 tcontext=system_u:object_r:mscan_etc_t:s0 tclass=dir permissive=0
```

**Comment by shawniverson** [comment]
```
type=AVC msg=audit(1595193118.248:1117): avc:  denied  { create } for  pid=1259 comm="php-fpm" name="sig.html.rules.bak" scontext=system_u:system_r:httpd_t:s0 tcontext=system_u:object_r:mscan_etc_t:s0 tclass=file permissive=0
```

**Comment by shawniverson** [comment]
```
type=AVC msg=audit(1595193192.729:1148): avc:  denied  { write } for  pid=6039 comm="php-fpm" path="/etc/MailScanner/rules/sig.html.rules.bak" dev="dm-0" ino=100743075 scontext=system_u:system_r:httpd_t:s0 tcontext=system_u:object_r:mscan_etc_t:s0 tclass=file permissive=0
```

**Comment by shawniverson** [comment]
Fixed in commit c224461a643cec70874314e732dfc88a9f271db9
